cogs/highlight.py

```diff
@@ -0,0 +1,209 @@
+import discord
+from discord.ext import commands
+from __main__ import send_cmd_help
+from cogs.utils.dataIO import dataIO
+import asyncio
+import aiohttp
+from datetime import datetime
+import os
+import itertools
+
+"""
+Cogs Purpose: To dm a user certain "highlight" words that they specify
+Requirements:
+
+Credit: This idea was first implemented by Danny (https://github.com/Rapptz/) but that bot is currently closed source.
+        So this is my own, and most definitely subpar, implementation of highlights
+"""
+
+def check_filesystem():
+
+    folders = ["data/highlight"]
+    for folder in folders:
+        if not os.path.exists(folder):
+            print("Highlight: Creating folder: {} ...".format(folder))
+            os.makedirs(folder)
+            
+    files = ["data/highlight/words.json"]
+    for file in files:
+        if not os.path.exists(file):
+            if "words" in file:
+                #build a default words.json
+                dict = {}
+                dict['guilds'] = []
+                dataIO.save_json("data/highlight/words.json",dict)
+                
+            print("Highlight: Creating file: {} ...".format(file))
+            
+class Highlight(object):
+    def __init__(self, bot):
+        self.bot = bot
+        self.highlights = dataIO.load_json("data/highlight/words.json")
+    
+    def _update_highlights(self, new_obj):
+        dataIO.save_json("data/highlight/words.json", new_obj)
+        self.highlights = dataIO.load_json("data/highlight/words.json")
+    
+    async def _sleep_then_delete(self, msg, time):

```

Comment:
This is nice 👌 .  I think this is great if you're adding/deleting from a channel and you don't want others to know your trigger words.